### ISSUE TITLE
Cannot set properties of null (setting 'checked')

### ISSUE THREAD

**Issue by ravingdragoon** [hktrpg/foundryVTT-speak-as#5]
Whenever I launch my world with Speak As enabled I get the following error message in the console. Is this due to no users being currently logged in? I don't tend to look at the console when I am actually running a game..

```
foundry.js:32606
TypeError: Cannot set properties of null (setting 'checked')
[Detected 2 packages: speak-as, lib-wrapper]
    at check (index.js:115)
    at index.js:101
    at Function._call (eval at <anonymous> (libWrapper-errors.js:8), <anonymous>:4:14)
    at Function.call (foundry.js:179)
    at ActorDirectory._render (foundry.js:2104)
    at async ActorDirectory._render (foundry.js:23500)
_call	@	VM237583:7
call	@	foundry.js:179
_render	@	foundry.js:2104
await in _render (async)		
🎁call_wrapped	@	libWrapper-wrapper.js:466
🎁Application.prototype._render#lib-wrapper	@	listeners.js:110
🎁Application.prototype._render#0	@	libWrapper-wrapper.js:161
_render	@	foundry.js:23500
(anonymous)	@	df-scene-enhance.js:5
fulfilled	@	df-scene-enhance.js:5
Promise.then (async)		
step	@	df-scene-enhance.js:5
(anonymous)	@	df-scene-enhance.js:5
__awaiter$2	@	df-scene-enhance.js:5
🎁Sidebar.prototype._render#df-scene-enhance	@	df-scene-enhance.js:5
🎁Sidebar.prototype._render#0	@	libWrapper-wrapper.js:161
render	@	foundry.js:2011
initializeUI	@	foundry.js:4487
setupGame	@	foundry.js:4384
await in setupGame (async)		
_initializeGameView	@	foundry.js:5213
_initializeView	@	foundry.js:5189
initialize	@	foundry.js:4286
await in initialize (async)		
🎁call_wrapped	@	libWrapper-wrapper.js:466
🎁libWrapperInit	@	libWrapper-api.js:695
await in 🎁libWrapperInit (async)		
🎁Game.prototype.initialize#0	@	libWrapper-wrapper.js:161
window.addEventListener.once	@	foundry.js:53613
```

**Comment by zeteticl** [comment]
@ravingdragoon 
Thankyou for report.
It wont effect the function.
But fixed in v1.2.1 
please check.

**Comment by ravingdragoon** [comment]
Bug fixed
